user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -368,18 +368,21 @@ define([
     },
 
     updateFolderButtons: function( _folderPath) {
-      const isRepoPath = !!_folderPath && isRepositoryPath(_folderPath);
+      const isRepositoryFolderPath = !!_folderPath && !isRepositoryRootPath(_folderPath) && isRepositoryPath(_folderPath);
```

Comment:
I think it would be better (at least easier to read) to test for `isRepositoryPath(.)` before `!isRepositoryRootPath(.)`.